### PR TITLE
docs: Add the onchain identifier guide

### PR DESCRIPTION
**Pull request by germartinez**
## Context
This PR:
- Moves this guide from Notion to our docs: https://safe-global.notion.site/Onchain-Identifiers-Docs-How-to-track-user-operations-dd159230b75c4994b59b05815a76c03b?pvs=74

### CODE REVIEW

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Track ERC-4337 transactions with an onchain identifier
+
+In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations `callData`.
+
+This guide summarizes all the steps included in the [Safe accounts with the Safe4337Module guide](./4337-safe-sdk.mdx), with an additional step to add the `onchainIdentifier` to your Safe transactions. If you already have the Relay Kit integrated, continue reading to learn how to get your onchain identifier, and feel free to jump straight to [step number 4](#add-your-onchain-identifier-to-the-user-operation-calldata) to learn how to use it afterward.
```
Comment (tanay1337) [inline pages/sdk/relay-kit/guides/onchain-identifier.mdx:7]:
```suggestion
This guide summarizes all the steps included in the [Safe accounts with the Safe4337Module guide](./4337-safe-sdk.mdx), with an additional step to add the `onchainIdentifier` to your Safe transactions. If you already have the Relay Kit integrated, continue reading to learn how to get your onchain identifier, and feel free to jump straight to [step number five](#add-the-onchainidentifier-to-the-calldata) to learn how to use it afterward.
```

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx [hunk 2]
```diff
@@ -0,0 +1,199 @@
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Track ERC-4337 transactions with an onchain identifier
+
+In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations `callData`.
```
Comment (tanay1337) [inline pages/sdk/relay-kit/guides/onchain-identifier.mdx:5]:
```suggestion
In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations (`callData`).
```

Comment (tanay1337) [reply pages/sdk/relay-kit/guides/onchain-identifier.mdx:5]: Should we also link to the User Operations entry in the Glossary?

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+This guide summarizes all the steps included in the [Safe accounts with the Safe4337Module guide](./4337-safe-sdk.mdx), with an additional step to add the `onchainIdentifier` to your Safe transactions. If you already have the Relay Kit integrated, continue reading to learn how to get your onchain identifier, and feel free to jump straight to [step number 4](#add-your-onchain-identifier-to-the-user-operation-calldata) to learn how to use it afterward.
+
+Please check the Safe4337Module linked guide for all the details regarding the steps summarized here.
```
Comment (tanay1337) [inline pages/sdk/relay-kit/guides/onchain-identifier.mdx:9]: Could potentially remove this statement as it's similar to the first statement of the last paragraph.

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx (hunk 2 above)
Comment (tanay1337) [reply pages/sdk/relay-kit/guides/onchain-identifier.mdx:5]: Re: my suggestion regarding the parenthesis, feel free to ignore if that's not accurate way to represent.

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+  Now, tracking all your transactions is possible by performing onchain queries looking for your identifier at the end of the `callData` field.
+
+</Steps>
+
+## Recap and further reading
+
+If you are not using the Safe\{Core\} SDK to submit transactions, the following GitHub repository may be helpful:
+
+- The [safe-onchain-identifiers](https://github.com/5afe/safe-onchain-identifiers) repository showcases where and how to add the identifier at the end of your Safe transactions data without using the Relay Kit and with a low-level detail. Check also the [specific code](https://github.com/5afe/safe-onchain-identifiers/blob/main/test/OnchainIdentifier.ts#L197-L217) where the identifier is concatenated to the `callData`.
```
Comment (tanay1337) [inline pages/sdk/relay-kit/guides/onchain-identifier.mdx:199]:
```suggestion
- The [safe-onchain-identifiers](https://github.com/5afe/safe-onchain-identifiers) repository showcases where and how to add the identifier at the end of your Safe transactions data without using the Relay Kit and with a low-level detail. Also check the [code](https://github.com/5afe/safe-onchain-identifiers/blob/main/test/OnchainIdentifier.ts#L197-L217) where the identifier is concatenated to the `callData`.
```

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx (hunk 2 above)
Comment (germartinez) [reply pages/sdk/relay-kit/guides/onchain-identifier.mdx:5]: Regarding the parenthesis, this should be the right option:
```
user operation's `callData`.
```

File: pages/sdk/relay-kit/guides/onchain-identifier.mdx (hunk 1 above)
Comment (germartinez) [reply pages/sdk/relay-kit/guides/onchain-identifier.mdx:7]: Good catch!

**Review by tanay1337 — APPROVED** [review]

**Pull request merged**